src/Microsoft.Azure.WebJobs.Host/Executors/IFunctionInstance.cs

```diff
@@ -27,5 +27,7 @@ public interface IFunctionInstance
     public interface IFunctionInstanceEx : IFunctionInstance
     {
         IServiceProvider InstanceServices { get; }
+
+        RetryContext RetryContext { get; }
```

Comment:
Again - this is a breaking change. You can't add new properties to public interfaces - it would break any existing implementations. I don't understand why you're adding this here.

Reply:
I though that IFunctionInstanceEx  is internal by some reason. Fixed.